create_docs.sh

```diff
@@ -24,4 +24,19 @@ for repo in adviser analyzer common lab package-extract python solver storages;
 	cp -r clones/${repo}/build/sphinx/html/* thoth/${repo}
 done
 
-
+# NOTE: If GITHUB_COMMIT is set,then thoth-station.github.io will be updated.
+if  [[ $GITHUB_COMMIT = "1" ]]; then
+    # Config: Script commit files on behalf of
+    git config --global user.name $GITHUB_USER
+    git config --global user.email $GITHUB_USER_EMAIL
+    rm -rf thoth-station.github.io
+    git clone git@github.com:thoth-station/thoth-station.github.io.git
+    cp -r thoth/* thoth-station.github.io/docs/developers/
+    cd thoth-station.github.io
+    git branch origin --delete feature/docs || true
+    git checkout -b feature/docs
+    git add .
+    git commit -m "Routine Docs Update"
+    git push origin feature/docs || exit 0
+    hub pull-request -l "bot" --no-edit || exit 0
```

Comment:
sure, that makes sense.
I was thinking if the prior pr hasn't been merged then it would cause the job to fails but this makes sense if it fails due to any other reason, we must be able to see the fails in openshift.
will change it.